inventory.example

```diff
@@ -0,0 +1,85 @@
+# based upon the following duffy provisioning
+#   [
+#        "n31.crusty",
+#        "d4de2418"
+#    ],
+#    [
+#        "n34.crusty",
+#        "d4de2418"
+#    ],
+#    [
+#        "n11.dusty",
+#        "d4de2418"
+#    ],
+#    [
+#        "n24.dusty",
+#        "d4de2418"
+#    ]
+#
+# and the duffy-openshift-3node-cluster.json topology file
+# the inventory file should look like this
+
+[masters]
+n31.crusty.ci.centos.org
+n34.crusty.ci.centos.org
+
+[OSEv3]
+n31.crusty.ci.centos.org
+n34.crusty.ci.centos.org
+n11.dusty.ci.centos.org
+n24.dusty.ci.centos.org
+
+[repo_host]
+n24.dusty.ci.centos.org
+
+[nodes]
+n31.crusty.ci.centos.org
+n34.crusty.ci.centos.org
+n11.dusty.ci.centos.org
+n24.dusty.ci.centos.org
+
+[masters:vars]
+openshift_override_hostname_check=true
+product_type=openshift
+ansible_ssh_user=root
+openshift_set_hostname=true
```

Comment:
Similar to above, if it is applied to all hosts, then it should just live under [OSEv3:vars]